src/plugins/workspace/public/plugin.test.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { of } from 'rxjs';
+import { coreMock } from '../../../core/public/mocks';
+import { ChromeNavLink } from 'opensearch-dashboards/public';
+import { WorkspacePlugin } from './plugin';
+
+describe('WorkspacePlugin', () => {
+  const plugin = new WorkspacePlugin();
```

Comment:
Prefer create plugin instance in every test cases. The different test case combination may cause unexpected test result if we share the plugin instance.